main/guides/orchestration/index.md

```diff
@@ -0,0 +1,19 @@
+# Orchestration Overview
+
+
+Orchestration is a core concept within the Agoric platform designed to empower entrepreneurs. It enables the creation of smart contracts that seamlessly interact with applications deployed on various blockchains. This composability allows for the development of user-centric applications that leverage the unique strengths of different blockchain ecosystems.
+The Agoric Orchestration API simplifies interactions between multiple networks, particularly those using the Inter-Blockchain Communication (IBC) protocol within Cosmos. The API acts as an abstraction layer, streamlining multi-step processes while maintaining requestor authority. Smart contracts involved gain no additional privileges. 
+Orchestration integrates with existing Agoric components (SwingSet, Cosmos modules) and introduces the vat-orchestration vat. This vat manages Inter-Chain Account (ICA) identities and connections to host chains, ensuring proper transaction authorization.
+The orchestration client handles asynchronous tasks and complex workflows, including those spanning multiple chains. This empowers smart contracts for actions like inter-chain staking and multi-hop transfers, facilitated by notifications from the transfer vat and IBC middleware updates. Orchestration simplifies complex cross-chain interactions within a secure and user-controlled environment on the Agoric platform.
+
+
+## Example API Usage
+
+```javascript
+const amount = coins(100, 'utia');
+const celestiaChain = E(orchestration).getChain('celestia');
+const icaCelestia = await E(celestiaChain).createAccount();
+await E(icaOsmosis).send(icaCelestia.receiver, amount);
+await E(timerService).delay(6000n);
+return E(icaCelestia).delegate(amount, celestiaValidator);
```

Comment:
**technical**

```suggestion
const icaCelestia = await celestiaChain.createAccount();
await icaOsmosis.transfer(icaCelestia.getAddress(), amount);
await E(timerService).delay(600n);
return icaCelestia.delegate(celestiaValidator, amount);
```

 - no `E` for chain objects, ica objects
 - `.receiver` -> `.getAddress()`
 - `.send()` only works within 1 chain. use `.transfer()` to do an IBC transfer across chains
 - `6000n` is not a typical time period. `600n` is more typical: 10 minutes.
 - **Note**: It's correct to use `E()` for `timerService`, but integration of `E` with the orchestration API is not yet available: https://github.com/Agoric/agoric-sdk/issues/9299
 - arguments to [delegate](https://agoric-sdk.pages.dev/interfaces/_agoric_orchestration.StakingAccountActions#delegate) go in the other order.

I'd really like us to use [snippets](https://github.com/Agoric/documentation/blob/main/CONTRIBUTING.md#importing-and-testing-code-snippets) of code that's tested in ci for examples such as this. Perhaps we can include https://github.com/Agoric/dapp-orchestration-basics as a git submodule and get snippets from there?

Reply:
Fixed as suggested.